libraries/helper.rb

```diff
@@ -35,6 +35,8 @@ def build_user_home
         File.join('/Users', node['omnibus']['build_user'])
       elsif windows?
         windows_safe_path_join(ENV['SYSTEMDRIVE'], 'Users', node['omnibus']['build_user'])
+      elsif node['platform_family'] == 'solaris2'
```

Comment:
I had that originally, specs failed all over the place... Might need to update Sugar a bit? I've never delved into its code.